compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/split_logicalobjfifos.mlir

```diff
@@ -0,0 +1,159 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(iree-amdaie-split-logical-objectfifos,cse)" --split-input-file --verify-diagnostics %s | FileCheck %s
```

Comment:
Don't see cse either I've confirmed this: maybe cherry https://github.com/nod-ai/iree-amd-aie/commit/ad6b693c943bf23cd7bf35decb7b870d10589822 (I've got some either minor changes there).

Reply:
Cool, thanks!

Reply:
I wasn't able to cherry-pick this commit. I tried to include most of your changes in the revision.

Reply:
Oh sorry, I wasn't sure if it would be possible to cherry-pick a commit. I think my mistake was rebasing your PR onto main, which meant that the collapsed commits included unrelated changes